src/main/kotlin/translations/lang/spellcheck/LangCommentTokenizer.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2020 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.translations.lang.spellcheck
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.impl.source.tree.LeafPsiElement
+import com.intellij.spellchecker.inspections.PlainTextSplitter
+import com.intellij.spellchecker.tokenizer.TokenConsumer
+import com.intellij.spellchecker.tokenizer.Tokenizer
+
+class LangCommentTokenizer : Tokenizer<LeafPsiElement>() {
+    override fun tokenize(element: LeafPsiElement, consumer: TokenConsumer?) {
```

Comment:
Looks like `consumer` is never null, you should be able to remove the nullability marker safely.

Reply:
I have removed this nullability marker.